### PR TITLE
Add arm64 migrations for mppp.

### PR DESCRIPTION
**Pull request by bluescarni**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bluescarni** [comment]
@conda-forge-admin, please rerender

File: recipe/migrations/osx_arm64.txt [hunk 1]
```diff
@@ -411,3 +411,5 @@ scitokens-cpp
 awscrt
 bc
 mypy
+mppp
+
```
Comment (xhochy) [inline recipe/migrations/osx_arm64.txt:415]:
```suggestion
```

**Comment by bluescarni** [comment]
@xhochy thanks a lot for the quick review!

I think I will need to add some logic to the build scripts for mppp to selectively disable some features on arm64. Do you have any pointer to the environment variables I could use to detect if the current build is targeting arm64?

**Comment by xhochy** [comment]
You can check in the build scripts with 
```
if [[ "${target_platform}" == "osx-arm64" ]]; then
```
In the meta.yaml you need `[osx and arm64]`.

**Comment by bluescarni** [comment]
> You can check in the build scripts with
> 
> ```
> if [[ "${target_platform}" == "osx-arm64" ]]; then
> ```
> 
> In the meta.yaml you need `[osx and arm64]`.

Cheers! Is there somewhere where I can read about the possible values of ``target_platform``? I need to be able to detect linux arm64 builds and ppc64el builds too in order to tweak the build process.

**Comment by xhochy** [comment]
You'll see it used in the `.ci_support` directory but I think it is undocumented. `linux-aarch64` and `linux-ppc64le` are the ones you are looking for.

**Comment by bluescarni** [comment]
@xhochy Ok, thanks a lot!

**Pull request merged**